README.md

```diff
@@ -71,6 +71,26 @@ For example, to install sage-attention (linux):
 
 However, you are highly recommended to first try without sage-attention since it will influence results, though the influence is minimal.
 
+**Docker**
+
+Build Docker Image:
+
+```bash
+sudo docker build -t framepack .
+```
+
+Run Container:
+
+```bash
+sudo docekr run -it \
+--rm \
+-p 7860:7680 \
+framepack \
+python demo_gradio.py
```

Comment:
Would recommend fixing the spelling of `docker` and then indenting the following lines and adding a `--name`.
```
sudo docker run \
  --name framepack \
  -it \
  --rm \
  -p 7860:7680 \
  framepack \
  python demo_gradio.py
```